### ISSUE TITLE
gogs: patches missing2: default paths for data

### ISSUE THREAD

**Issue by mckaygerhard** [Martchus/PKGBUILDs#38]
after installing (maybe due documentation are still in progress) default paths cuses lot of errors like

```
[ERROR] [...rc/runtime/asm_386.s:498 call16()] GenerateRandomAvatar: Create: open /var/lib/gogs/data/avatars/1: permission denied
```

this due default dir are created in install but subdirs are missing like : 
* data/attachements
* data/avatars
* data/sessions
* certs

**Comment by Martchus** [comment]
It should be sufficient to create `/var/lib/gogs/data`. Likely the directories permissions aren't correctly in your case (so gogs fails to create the subdirs). However, the install script should actually ensure the directory is owned by `gogs`.

Here is how the permissions look at my server:
```
sudo ls -l /var/lib | grep gogs
drwx------ 6 gogs      gogs        4096 23. Aug 19:57 gogs
```

**Comment by mckaygerhard** [comment]
no its not enough! for avatars and you create the /etc/gogs/app.ini as the custon read by default file.. (umm i should reopen the other issue)

i have the right permission, inclusivelly, i installed a ARCH machine and use your pkgbuild (with help of a friend of my) and happened the same..  see the other issue (i'll reopen)

**Comment by Martchus** [comment]
Ok, I now ensure those dirs are present: https://github.com/Martchus/PKGBUILDs/commit/1d95d55ae7e1a144d7477bec35921c92e4d017dc

Regardless whether it is actually required, this changes shouldn't not cause any trouble, so why not.

**Comment by mckaygerhard** [comment]
ahhh! there's a previous issue about that: #25 so now it's solved with  1d95d55
